### PR TITLE
Added an helm chart for an easy installation of o-neko

### PR DESCRIPTION
**Pull request by steffenkuche**

### CODE REVIEW

File: helm/oneko/templates/o-neko-deployment.yaml [hunk 1]
```diff
@@ -42,7 +44,9 @@ spec:
               port: 8080
             initialDelaySeconds: 5
             periodSeconds: 5
-            timeoutSeconds: 5
+            timeoutSeconds: 30
+      imagePullSecrets:
+        - name: dockersubshell
```
[the remark this answers is not in the document]
Comment (philmtd) [reply helm/oneko/templates/o-neko-deployment.yaml]: No it doesn't. It's publicly accessible here: https://hub.docker.com/r/subshellgmbh/o-neko and the `latest-dev` tag is always updated with the latest master branch version.

File: helm/oneko/templates/o-neko-service.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ metadata:
 spec:
   selector:
     o-neko: o-neko
+  type: LoadBalancer
```
[the remark this answers is not in the document]
Comment (steffenkuche) [reply helm/oneko/templates/o-neko-service.yaml]: a clusterIP service is sufficient as well

File: helm/oneko/values.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+kubernetesPort: 6443
+ingressClass: traefik
+host: oneko.subshell.cloud
```
[the remark this answers is not in the document]
Comment (steffenkuche) [reply helm/oneko/values.yaml]: that's why this configuration is placed in the values.yaml. For an easy adaption. This is as good as example.com.

File: helm/oneko/templates/o-neko-deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (steffenkuche) [reply helm/oneko/templates/o-neko-deployment.yaml]: yeah, actual this is not necessary

File: helm/oneko/values.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (schwerlaut) [reply helm/oneko/values.yaml]: I agreed with @reinkem here. This should not be part of the code base here.

**Pull request merged**